h/search/connection.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+from elasticsearch_dsl import connections
+
+# TODO: Temporary hard-coding of ELASTICSEARCH_URL: this should come from settings
+ELASTICSEARCH_URL = os.environ.get("ELASTICSEARCH_URL", "http://localhost:9201")
```

Comment:
As TODO notes, this hard-coded thing is in place until refactoring of settings makes this path more clear. I'm hoping this environment-var approach will allow Jenkins builds to work (assuming the `ELASTICSEARCH_URL` is present there).